nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -108,6 +108,18 @@ providers:
                 - python3-virtualenv
                 - unbound
               users:
+                - name: op
```

Comment:
will root work here?

Reply:
I cannot really say, we need to give it a try.

Reply:
Ok, I just took a look. It's too complicated with `root`.